src/studio/kdb/Config.java

```diff
@@ -444,6 +444,15 @@ public void setShowServerComboBox(boolean value) {
         save();
     }
 
+    public int getResultTabsCount() {
+        return Integer.parseInt(p.getProperty("resultTabsCount","5"));
```

Comment:
For table, list and dictionary results, we open 2 tabs, one with the text and one with the table view. With a default limit of 5 tabs, after the user does 3 queries there will be 2 table tabs, 2 text tabs and one tab left over from the remaining query that loses its counterpart.
Maybe we should only ever allow even numbers for the limit (although that can still cause odd behavior if the user does mixed text-only and table/text queries), or treat the text and the table tab of a query as a single unit for the purpose of this limit?

Reply:
Can't think of a straightforward way to consider the text and the table tab as a single unit - so i suggest to go with the current behavior for now as it's simple and can potentially improve over time?

Reply:
It's OK to have the current behavior but at the very least having an odd number as the default will stick out to any user who does a lot of table queries which is arguably the most important use case for studio.

Reply:
ok so I can make the default 6?

Reply:
changed to 6